addons/pylons/functions/fnc_showDialog.sqf

```diff
@@ -93,6 +93,7 @@ GVAR(comboBoxes) = [];
     {
         _combo lbAdd getText (configFile >> "CfgMagazines" >> _x >> "displayName");
         _combo lbSetData [_forEachIndex + 1, _x];
+        _combo lbSetTooltip [_forEachIndex + 1, getText (configFile >> "CfgMagazines" >> _x >> "descriptionShort")];
```

Comment:
Could save `configFile >> "CfgMagazines" >> _x` in a variable.

Reply:
```suggestion
        private _description = getText (configFile >> "CfgMagazines" >> _x >> "descriptionShort");
        _combo lbSetTooltip [_forEachIndex + 1, _description];
```

Reply:
I thought about it, but we are just filling a dozen entries here, and only when the UI opens. very low perf impact code here.
jonpas, why? just readability? it already says descriptionShort there so why say it twice?

Reply:
I meant the config lookup - `private _magazineConfig = configFile >> "CfgMagazines" >> _x;`. But yeah, doesn't really matter.

Reply:
Readability, less wide lines. But I mostly made a suggestion as an addition to mharis' comment. :p

Reply:
> I meant the config lookup

I know